content/guides/outliers.md

```diff
@@ -80,75 +80,51 @@ There are two different outlier detection algorithms you can use on your data: D
+A natural way to group together hosts that are behaving similarly is to use a clustering algorithm. We use [DBSCAN](https://en.wikipedia.org/wiki/DBSCAN), a popular density-based clustering algorithm, for this purpose. DBSCAN works by greedily agglomerating points that are close to each other. Clusters with few points in them are considered outliers.
 
-We use a simplified form of DBSCAN to detect outliers; any point that is not in the largest cluster will be considered an outlier. We consider each host to be a d-dimensional vector, where d is the number of values in the time series. Any hosts that are within a distance of `alpha x threshold` of each other will get clustered together.
+Traditionally, DBSCAN takes: 1) a parameter 𝜀 that specifies a distance threshold under which two points are considered to be close; and 2) the minimum number of points that have to be within a point’s 𝜀-radius before that point can start agglomerating. The image below shows an example of DBSCAN in action on points in the plane. There are two clusters. The large points had enough close neighbors to agglomerate those points, while the small colored points did no agglomerating themselves but are within the 𝜀-radius of a large point. The points in black are the outliers.
 
-The threshold is calculated as follows. We create a new median time series by taking the median of the values from the existing time series at every time point. Then we calculate the distance between each host and the median series. The threshold is the median of those distances.
-
-In pseudocode:
-
-~~~ python
-    median_series = pointwise_median(series_list)
-    dists = [dist(series,median_series) for series in series_list]
-    threshold = median(dists)
-~~~
+<img src="/static/images/outliers/outliers-dbscan-2d.png" style="width:100%; border:1px solid #777777"/>
 
 #### Parameters
 
-DBSCAN accepts the following parameters:
+We use a simplified form of DBSCAN to detect outliers on time series. (We consider each host to be a point in d-dimensions, where d is the number of values in the time series.) Any point can agglomerate, and any point that is not in the largest cluster will be considered an outlier.
 
-* **alpha** *(default: 3.0)*: Sets the constant by which the threshold is multiplied.
+We set the initial distance threshold as follows. We create a new median time series by taking the median of the values from the existing time series at every time point. Then we calculate the distance between each host and the median series. The threshold is the median of those distances.
 
-For example, to use DBSCAN with an alpha of 3 on system load for a pool of Cassandra workers, we'd use:
+The only parameter we take is: alpha. This sets the constant by which the initial threshold is multiplied to yield DBSCAN’s distance parameter 𝜀. Here is DBSCAN with an alpha of 3.0 in action on a pool of Cassandra workers:
```

Comment:
I'm being a complete nitpick, but... Might alpha be offset with code formatting and the first two sentenced combined? I just think it might read a little smoother that way.

> The only parameter we take is `alpha`, the constant by which ... Here is DBSCAN with an `alpha` of 3.0 ...